openslides/motions/static/js/motions/site.js

```diff
@@ -1124,6 +1125,43 @@ angular.module('OpenSlidesApp.motions.site', [
             $scope.motion = Motion.get(motion.id);
             MotionComment.populateFields($scope.motion);
         });
+        $scope.projectionModes = [
+            {mode: 'original',
+            label: 'Original version'},
+            {mode: 'changed',
+            label: 'Changed version'},
+            {mode: 'diff',
+            label: 'Diff version'},
+            {mode: 'agreed',
+            label: 'Resolution'},
+        ];
+        $scope.projectionMode = function getProjectedMode() {
+            var projectedIds = motion.isProjected();
+            if (projectedIds.length) {
+                var element = _.find(Projector.get(projectedIds[0]).elements, function (element) {
+                    return element.name == 'motions/motion' && element.id == motion.id;
+                });
+                var modeName = element.mode || 'original', mode;
+                _.forEach($scope.projectionModes, function (mode_) {
+                    if (mode_.mode == modeName) {
+                        mode = mode_;
+                    }
+                });
+                return mode || $scope.projectionModes[0];
+            } else {
+                return $scope.projectionModes[0];
+            }
+        }();
+        $scope.setProjectionMode = function (mode, event) {
+            $scope.projectionMode = mode;
+
+            var projectedIds = motion.isProjected();
+            _.forEach(projectedIds, function (id) {
+                motion.project(id, mode.mode);
+            });
+
+            event.stopPropagation();
```

Comment:
Is this necessary? I always thought Angular cares about handling ng-click and thelike properly.